src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -406,6 +440,19 @@ private void runGenotypeGVCFSAndAssertSomething(String input, File expected, Lis
         }
     }
 
+    private void runGenotypeGVCFSAndAssertCount(String input, List<String> additionalArguments, Integer count, BiConsumer<VariantContext, Integer> conditionOnCount, String reference) {
+        final File output = runGenotypeGVCFS(input, null, additionalArguments, reference);
+        Assert.assertTrue(output.exists());
+
+        final List<VariantContext> actualVC = VariantContextTestUtils.getVariantContexts(output);
+        Assert.assertTrue(actualVC.size() > 0);
+        try {
+            assertCountForEachElementInList(actualVC, count, conditionOnCount);
+        } catch (final AssertionError error) {
+            throw error;
```

Comment:
Why catch and immediately re-throw?

Reply:
This was the easiest way to debug, but I can take it out.

Reply:
How did you let me get away with no `final`s on this method???